SpatialGDK/Source/SpatialGDK/Public/Schema/UnrealObjectRef.h

```diff
@@ -45,6 +45,16 @@ struct FUnrealObjectRef
 		return FString::Printf(TEXT("(entity ID: %lld, offset: %u)"), Entity, Offset);
 	}
 
+	FORCEINLINE FUnrealObjectRef GetLevelReference() const
+	{
+		if (Path->Equals(TEXT("PersistentLevel")))
+		{
+			return *this;
+		}
+
+		return Outer->GetLevelReference();
```

Comment:
This will crash for an UnrealRef without an outer.

Reply:
monkaS